package.json

```diff
@@ -47,7 +48,7 @@
     "eslint-plugin-simple-import-sort": "^8.0.0",
     "jest": "^29.7.0",
     "lint-staged": "^13.3.0",
-    "osml-cdk-constructs": "^1.6.0",
+    "osml-cdk-constructs": "file:lib/osml-cdk-constructs",
```

Comment:
You might want to undo this and update to the latest version.

Reply:
Yep! just part of testing before the CDK was published.  Good call-out!